pkg/minikube/cruntime/docker.go

```diff
@@ -91,8 +91,10 @@ func (r *Docker) DefaultCNI() bool {
 
 // Available returns an error if it is not possible to use this runtime on a host
 func (r *Docker) Available() error {
-	_, err := exec.LookPath("docker")
-	return err
+	if _, err := exec.LookPath("docker"); err != nil {
+		return errors.New("docker not found in $PATH. please install docker or try adding it to the $PATH")
```

Comment:
this is not the right place to add a solution message

Reply:
Can you suggest where should I put this message?

As you said the `open an issue` shouldn't appear, I'll change the log function from error -> withMessage.